src/node_modules/editor/index.js

```diff
@@ -33,6 +33,24 @@ const output = CodeMirror.fromTextArea(
   Object.assign({}, opts, { readOnly: true })
 )
 
+async function clearDocumentWindow () {
+  await sandbox.webContents.executeJavaScript(`
+    try {
+      eval(\`(function() { document.body.innerHTML='' })()\`)
```

Comment:
Do we still need this if we do `sandbox.reload()` on line 51?

Reply:
Yuck ..Not really .Missed it.Resubmitting

Reply:
Yes because the document window be cleared otherwise.

Reply:
👍 I say we merge into master and try it out.

Reply:
sure thanks